### DISCUSSION TITLE
5.1.0版本，自定义延时时间，延时时间大于0的情况下，无法监听到消息

### DISCUSSION THREAD

**Discussion by blukhand** [apache/rocketmq#7616] [Q&A]
问题描述： 集群部署的mq。版本为5.1.0。 使用示例代码做延时消息发送，延时时间大于0的情况下，无法监听到消息。
相关：  
1.  Topic已经设置为延时主题
2. 如果 时延设置为0或者负数，可以立刻被监听到，符合文档的描述。说明Topic没有设置错。
3. 在本机起了单节点的RocketMq，相同代码，修改ip端口，可以正常发送延时消息。应该客户端代码也没有问题。

想请问下，这个可能是什么原因导致的，排查方向是怎么样？
请问延时消息是否对 server端的部署配置有要求？
延时中的消息（未到发送时间），可以通过什么途径查询到？

**Comment by onejimmyboy** [comment]
发送延迟消息的时候可以看一下store.log的日志，还有服务端和客户端的时间也要保持一致
